modules/gltf/gltf_document.cpp

```diff
@@ -4803,9 +4803,10 @@ Error GLTFDocument::_parse_animations(Ref<GLTFState> state) {
 		Array samplers = d["samplers"];
 
 		if (d.has("name")) {
-			const String name = d["name"];
-			if (name.begins_with("loop") || name.ends_with("loop") || name.begins_with("cycle") || name.ends_with("cycle")) {
+			String name = d["name"];
+			if (name.begins_with("loop") || name.ends_with("loop") || name.begins_with("cycle") || name.ends_with("cycle") || name.find(" -loop") > 0) {
 				animation->set_loop(true);
+				name = name.replace(" -loop", "");
```

Comment:
` -loop` with the extra space. Is this intended?

Reply:
it is

Reply:
some examples of my use case:
![image](https://user-images.githubusercontent.com/1776044/127755540-f50829be-79d7-4cc8-a563-674f3615a8b9.png)

Reply:
Does Blender's search only work with whole words or something? It seems strange that `loop` wouldn't match if there is no space before the hyphen.

Reply:
space is a personal taste, because of you have many arguments then you separate them with spaces, ie `Idle -loop -autostart`

Reply:
We should refactor the import hints to allow this with all hints, rather than just `-loop|. Behavior should be consistent and documented.

Reply:
I would be surprised if it didn't work with a space. It will just strip the suffix and your imported name will be e.g. `"RacingIdle "`. Godot might then strip the space.

Either way, personal tastes shouldn't put limitations on what others can do so this shouldn't include the space in the test.

Reply:
Does blender have a checkbox somewhere to set an animation as a 'looping animation' might be a bit cleaner? 
It might be better than relying on the naming, its not even supported in all imported formats to set the animation name to have -loop in it

Reply:
The F-Modifier flag seems to do this, but IDK if it's going to be in an exported file.
![image](https://user-images.githubusercontent.com/748770/127854801-1262fcff-0ad7-49d3-b99f-3a028d285e9c.png)

Reply:
another reason to have the space and dash is to avoid backward compatibility breakage, since up until now starting or ending the name of the animation with `loop` will make it loop, but will not remove it from the animation name. I think we could use custom properties for looping, but that is not a very friendly workflow for people exporting assets (inho is even less friendly than forcing to rename the animations to end with loop)

Reply:
My view is that this pr and the enhancement proposal for the extra space should be separated and of course have a written proposal.